calculus/probability_density_functions/java/Gaussian.java

```diff
@@ -0,0 +1,9 @@
+public class Gaussian {
+    public static double pdf(double x) {
+        return Math.exp(-x*x / 2) / Math.sqrt(2 * Math.PI);
```

Comment:
This seems like a simplification of http://support.minitab.com/en-us/minitab-express/1/help-and-how-to/basic-statistics/probability-distributions/how-to/probability-density-function-pdf/methods-and-formulas/methods-and-formulas/

Reply:
Yes, it is the normal distribution (also called Gaussian distribution) special case when mu = 0 and sigma = 1. There's also common case in the PR - overloaded function with mu and sigma given as parameters.